src/sonic-yang-mgmt/tests/libyang-python-tests/test_sonic_yang.py

```diff
@@ -329,12 +329,12 @@ def test_xlate_rev_xlate(self, sonic_yang_data):
         jIn = json.loads(jIn)
         numTables = len(jIn)
 
-        syc.loadData(jIn, debug=True)
```

Comment:
With debug=True, we create json files with yang and config db format.  Any problem in creating these files, while testing. Sometime they are really useful when test case fails.

Reply:
I was facing issue with compilation because the test fails with permission errors while creating the debug file.